picasa.json

```diff
@@ -0,0 +1,23 @@
+{
+    "homepage": "https://en.wikipedia.org/wiki/Picasa",
+    "version": "3.9.141.Build-259",
+    "description": "Picasa is a discontinued image organizer and image viewer for organizing and editing digital photos, plus an integrated photo-sharing website by Google",
+    "license": "Freeware",
+    "hash": "482c1a547d8d3aa25ee446d30ea986de63ef8c8d68b8d1109dd3d9b714e73e08",
+    "url": "https://github.com/raisercostin/raiser-scoop-bucket/releases/download/picasa-3.9.141.259/picasa39-setup-3.9.141.259.exe#dl.7z",
```

Comment:
Why is this pointing to a random release in your repo? Does this need to be in scoop-extras if it's already in your personal bucket?

Reply:
The problem is that the picasa is discontinued and the binary is not to be found easily on internet.
I was thinking that there are still people that still use it.
I uploaded a version that shouldn't contain any malware.